### ISSUE TITLE
Migrate Google Code releases to here

### ISSUE THREAD

**Issue by 10by10pixel** [scottdraves/electricsheep#34]
Downloading for Windows links to a 404 thanks to the shut down Google Code we need to upload the compiled binaries here as releases.

Relevant Documentation: https://help.github.com/articles/creating-releases/

**Comment by 2ajoyce** [comment]
I wanted to install Electric Sheep and ran into this 404 error. Any chance that those binaries will be up in the near future?

**Comment by thewebkid** [comment]
+1. I have had this as my screensaver for 10 years and I have a new laptop with a really awful screensaver. I really don't want to try to build my own exe from the repo :)

...but I will if the bits don't materialize soon!

**Comment by sildur** [comment]
I made a fork and a release, hoping I could make a pull request for that, but seems not possible. I tagged 1d90bc4e9b927f337a7543178b0bfbe8ea2ec919 as the release, as that was the one which bumped Mac's version to 2.7b34 (previous one, 1d90bc4e9b927f337a7543178b0bfbe8ea2ec919,  bumped everything else's versions) and I wanted to include both Mac and Windows versions into the release.

I got the Mac version from the Downloads section in Electric Sheep's website, and the Windows version from archive.org (https://electricsheep.googlecode.com/files/electricsheep-2.7b34.exe)

Anyways, if someone knows how to make a PR from a release, I'd be glad to do one. In the meantime, here is my release: https://github.com/sildur/electricsheep/releases

**Comment by scottdraves** [comment]
thanks for taking the initiative.  good idea i got what i think is an even more recent version (b34c) and updated the link.  can you please test?  i don't have a windows machine...

**Comment by scottdraves** [comment]
http://electricsheep.org/node/49 and http://electricsheep.org/node/19

**Comment by 10by10pixel** [comment]
@scottdraves Help menu points to a broken link

**Comment by sildur** [comment]
@scottdraves I tested it, and looks good. Everything works, and the video expands to cover all the screen. This is a Windows 11 machine, by the way. I added a new release, and made it point to the v2.7b36 commit (e08fa59f7aa22fe6d4eaac30ef049bdcd739e393), as made more sense, since the Mac version is v2.7.b36.

By the way, the windows download link here http://electricsheep.org/node/19 points to https://d100rc88eim93q.cloudfront.net/electricsheep-2.7b34.exe, which returns "Access Denied". The right link would be http://d100rc88eim93q.cloudfront.net/electricsheep-2.7b34c.exe
